TagV3.0_U575VGT/Core/Inc/Sensor Inc/LightSensor.h

```diff
@@ -143,5 +145,5 @@ HAL_StatusTypeDef LightSensor_get_data(LightSensorHandleTypedef *light_sensor);
 HAL_StatusTypeDef LightSensor_set_data_rate(LightSensorHandleTypedef *light_sensor, ALSIntegrationTime int_time, ALSMeasureTime meas_rate);
 HAL_StatusTypeDef LightSensor_get_part_id(LightSensorHandleTypedef *light_sensor, ALSPartIDRegister *dst);
 HAL_StatusTypeDef LightSensor_get_manufacturer(LightSensorHandleTypedef *light_sensor, ALSManufacIDRegister *dst);
-
+HAL_StatusTypeDef LightSensor_thread_entry(ULONG thread_input);
```

Comment:
I believe that threads are expected to return void, so im not sure if returning a HAL status def is going to do something here